### ISSUE TITLE
Implementation breaks accessibility actions menu to delete cells

### ISSUE THREAD

**Issue by staticnz** [CEWendel/SWTableViewCell#218]
There are conflicts with the - (void)tableView:(UITableView)tableView commitEditingStyle:(UITableViewCellEditingStyle)editingStyle forRowAtIndexPath:(NSIndexPath*)indexPath method.

If this method is implemented, there are issues with the swiping to delete where it conflicts with the system method. However without the system method, users with Accessibility turned on and cannot swipe, are unable to action deletion.

Some more info: http://stackoverflow.com/questions/20786827/attempting-to-set-a-swipe-to-delete-cell-when-we-already-have-one-that-doesn

**Comment by jlubeck** [comment]
Hey @staticnz where you ever able to solve this? I ran into the same issue...
